### PR TITLE
Antonina Turieva hw 2

### PR DESCRIPTION
**Pull request by AntTurieva**
Homework 2: arithmetic operations, comparison operators and logical operations

### CODE REVIEW

File: src/main/java/hw_2_technical_task/App.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package hw_2_technical_task;
+
+public class App {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+        System.out.println(x + "" + y);
```
Comment (AntBelyi) [inline src/main/java/hw_2_technical_task/App.java:9]: THere is should be the output in one line
Screenshot_5

**Comment by AntTurieva** [comment]
Fixed, but apparently I have problems with the Internet, then nothing was sent, now there are a lot of requests. Sorry, I accidentally deleted your comment

File: src/main/java/hw_2_technical_task/App.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package hw_2_technical_task;
+
+public class App {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println((x + y) + " and " + x + y);
```
Comment (AntBelyi) [inline src/main/java/hw_2_technical_task/App.java]: You can use just System.out.println(x + y + " and " + x + y);

without additional (...) around x + y

**Comment by AntBelyi** [comment]
Please create only hw_2 folder and add there Task1 and Task2
Screenshot_13

File: src/main/java/hw_2_technical_task/App.java (hunk 2 above)
Comment (AntTurieva) [reply src/main/java/hw_2_technical_task/App.java]: fixed, thanks

Comment (AntTurieva) [reply src/main/java/hw_2_technical_task/App.java]: fixed this too https://github.com/AntBelyi/ALevel_automation/pull/16#issuecomment-2188864783

**Pull request merged**